app/models/manageiq/providers/amazon/cloud_manager/virtual_template.rb

```diff
@@ -1,3 +1,4 @@
 class ManageIQ::Providers::Amazon::CloudManager::VirtualTemplate < ::ManageIQ::Providers::CloudManager::VirtualTemplate
-  validates :cloud_subnet_id, :availability_zone_id, :ems_ref, :presence => true
+  validates :cloud_network_id, :availability_zone_id,
```

Comment:
i think validating presence of the actual object vs the id is better
see [this post](http://railsguides.net/belongs-to-and-presence-validation-rule1/) for a good explanation